week3/homework/src/myFunctions.js

```diff
@@ -0,0 +1,80 @@
+const fs = require('fs');
+function createToDo(request, response) {
+  fs.readFile('./todos.json', 'utf8', (error, contents) => {
+    if (error) {
+      response.status(500).send('Whats wrong with you');
+    } else {
+      const updatedContent = JSON.parse(contents);
+      updatedContent.push(request.body);
+      const updatedNewContent = JSON.stringify(updatedContent);
+      fs.writeFile('./todos.json', updatedNewContent, err => {
+        if (err) throw err;
+      });
+      response.send(updatedNewContent);
+    }
+  });
+}
+
+function readToDo(request, response) {
+  fs.readFile('./todos.json', 'utf8', (error, file) => {
+    if (error) {
+      response.status(500).send('There is no toDoList file.');
+      return 'An Error Occured';
+    } else {
+      const id = request.params.id;
+      response.json(JSON.parse(file)[id - 1]);
+    }
+  });
+}
+
+function deleteToDo(request, response) {
+  fs.readFile('./todos.json', 'utf8', (error, contents) => {
+    if (error) {
+      response.status(500).send('Whats wrong with you');
+    } else {
+      const updatedContent = JSON.parse(contents);
```

Comment:
This code seems to be repeated in several places, violating the DRY principle. I would suggest that you put it in a separate function so that it can be reused.